src/styles.css

```
@@ -709,3 +709,83 @@ footer a:hover{text-decoration:underline}
+  gap: 20px;
+}
+
+/* Tablet Devices */
+@media screen and (max-width: 992px) {
+
+  .nav-inner {
+    gap: 16px;
+  }
+
+  .nav-links,
+  #mobileMenu {
```

Stylelint (17.12.0)

[error] Expected id selector "`#mobileMenu`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)